scripts/mpv-config

```diff
@@ -12,7 +12,10 @@ if test -f "$BUILD"/mpv_options ; then
 fi
 
 if "$BUILD"/scripts/test-libmpv; then
-    OPTIONS="-Dc_link_args='-Wl,-Bsymbolic'"
+    platform=$(uname | tr '[:upper:]' '[:lower:]')
+    case "$platform" in
+    *bsd*|*linux*|*solaris*) OPTIONS="-Dc_link_args='-Wl,-Bsymbolic'" ;;
```

Comment:
shouldn't this append instead of overwrite?

Reply:
What do you mean? `OPTIONS` is empty otherwise (maybe this should have a better name).

Reply:
nvm, I misinterpreted the reading of `mpv_options` above